crates/bevy_reflect/bevy_reflect_derive/src/registration.rs

```diff
@@ -3,14 +3,15 @@
 use bit_set::BitSet;
 use proc_macro2::Ident;
 use quote::quote;
-use syn::{Generics, Path};
+use syn::{Generics, Path, Type};
 
 /// Creates the `GetTypeRegistration` impl for the given type data.
 pub(crate) fn impl_get_type_registration(
     type_name: &Ident,
     bevy_reflect_path: &Path,
     registration_data: &[Ident],
     generics: &Generics,
+    field_types: &Vec<Type>,
```

Comment:
Nit: I don't think we need to add this. We should be able to get the generic param info from `generics`, right?

Reply:
Oh I think I see why we do it this way. This ensures we only add the bound to actively used (non-`#[reflect(ignore)]`) types.

Reply:
Yes exactly; we want to ignore generics that are inactive